packages/espressocash_app/lib/app/screens/authenticated/investments/investments_screen.dart

```diff
@@ -62,7 +62,7 @@ class _InvestmentsScreenState extends State<InvestmentsScreen> {
                   SliverToBoxAdapter(child: SizedBox(height: 24)),
                   SliverToBoxAdapter(child: PopularCryptoHeader()),
                   PopularTokenList(),
-                  SliverToBoxAdapter(child: SizedBox(height: 12)),
+                  SliverToBoxAdapter(child: SizedBox(height: 40)),
```

Comment:
Don't use hardcoded value for fixing system bars offset. Doesn't `SafeArea` help?

Reply:
SafeArea seems to overpad the bottom area. I've updated using MediaQuery